index.html

```diff
@@ -84,53 +106,41 @@ <h3>Costume</h3>
+  <section class="featured-display">
+    <h2>Main Program</h2>
+    <ul class="program">
+      <li class="program-display">
+        <i class="fas fa-user-alt"></i>
+        <h3 class="program-h3">Cast</h3>
+        <p class="program-p1">Meet all your favorite actors and the crew that featured in all parts of the movie.</p>
       </li>
-      <li>
-        <img src="images/checkers.jpg" alt="checkers" class="checkers">
-        <img src="images/sophie-turner.jpg" alt="sophie-turner" class="featured-image">
-        <h3>Sophie Jonas</h3>
-        <p class="p-colour">English actress</p>
-        <p class="p-spacing">Sophie Belinda Jonas is an English actress. She made her acting debut as Sansa Stark on the HBO epic fantasy television series Game of Thrones, for which she received a Primetime Emmy Award nomination for the Outstanding Supporting Actress in a Drama Series in 2019</p>
+      <li class="program-display">
+        <i class="far fa-eye"></i>
+        <h3>Exhibition</h3>
+        <p>See all the real props and materials used for the movie and how they were created by the crew</p>
       </li>
-      <li>
-        <img src="images/checkers.jpg" alt="checkers" class="checkers">
-        <img src="images/lena-headey.jpg" alt="lena-headey" class="featured-image">
-        <h3>Lena Headey</h3>
-        <p class="p-colour">English Actor</p>
-        <p class="p-spacing">Lena Kathren Headey is an English actress. She gained international recognition and acclaim for her portrayal of Cersei Lannister on the HBO epic fantasy drama series Game of Thrones, for which she received five Primetime Emmy Award nominations and a Golden Globe Award nomination.</p>
+      <li class="program-display">
+        <i class="far fa-comment-dots"></i>
+        <h3 class="program-h3">Forum</h3>
+        <p class="program-p">Have the time to speak to the writers and crew about your favorite scene or scenes that confused you</p>
       </li>
-      <li>
-        <img src="images/checkers.jpg" alt="checkers" class="checkers">
-        <img src="images/peter-dinklage.jpg" alt="peter-dinklage" class="featured-image">
-        <h3>Peter Dinklage</h3>
-        <p class="p-colour">American actor</p>
-        <p class="p-spacing">Peter Hayden Dinklage is an American actor and producer. He received acclaim for portraying Tyrion Lannister on the HBO television series Game of Thrones, for which he won the Primetime Emmy Award for Outstanding Supporting Actor in a Drama Series a record four times.</p>
+      <li class="program-display">
+        <i class="fas fa-tshirt"></i>
+        <h3 class="program-h3">Costume</h3>
+        <p>Try creating your own props and storyline and dress up as your favorite actors.</p>
       </li>
     </ul>
+    <h4><a href="#">Game Of Thrones
+      Fans Summit
+      2021</a></h4>
+  </section>
+
+  <section class="featured-speakers">
+    <h2>Featured Speakers</h2>
+    <ul class="speakers">
+    </ul>
   </section>
```

Comment:
- [ ] Let us try to get closer to the design with it's font-size and the colors. Look for the icons, and the text. Moreover, our `Main program` title should be underlined with red color line.

----------------------------------------Original---------------------------------------

![rev3](https://user-images.githubusercontent.com/55635977/133493656-47cf5ca9-9958-46d0-a878-7159cfdc7a54.png)

----------------------------------------Yours---------------------------------------

![rev4](https://user-images.githubusercontent.com/55635977/133493813-fc1d97ca-4f73-4513-8085-95e500e5dfb7.png)

Reply:
This issue is resolved! ✔️